specs/ec/ec.tla

```diff
@@ -10,118 +17,158 @@ Range(f) ==
 
 (* --algorithm ExpectedConsensus
 variables
-    NumMiners=5;
-    miners = 1..NumMiners;
-    MaxRounds=10;
-    MaxLeaders = 3;
-    rounds = 1..MaxRounds;
-    miner_rounds = [miner \in miners |-> 0];
-    leaders = <<{}>>;
+    miner_rounds = [miner \in miners |-> 2];
```

Comment:
Why do you initially set to 2?

Reply:
Because we need to be able to inspect the previous round in order to choose a tipset to mine on. We arbitrarily select a single first leader in round 1, so round 2 is the first real round. (This could be made more sophisticated later, but starting with one leader makes the implementation easier for now.)

Reply:
Why do you pick a single one in round 1 as opposed to just running the same process as for other rounds (starting with some genesis tipset eg)? It's not immediately clear to me from the code...